### PR TITLE
feat: add an add to GitHub button on an encalve

### PR DESCRIPTION
**Pull request by h4ck3rk3y**
First frontend PR, please review properly

Note that the instance id and api key should fill up when in cloud!

### CODE REVIEW

**Comment by h4ck3rk3y** [comment]
Uploading Screen Recording 2024-03-05 at 1.18.40 PM.mov…

File: enclave-manager/web/packages/app/src/emui/enclaves/components/widgets/RunViaCIButton.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    return (
+      <Button isLoading={true} colorScheme={"yellow"} leftIcon={<FiGithub />} size={"sm"} {...buttonProps}>
+        Add GitHub Action
+      </Button>
+    );
+  }
+
+  if (enclave.starlarkRun.isErr) {
+    return (
+      <Tooltip label={"This enclave didn't really load"}>
```
Comment (tedim52) [inline enclave-manager/web/packages/app/src/emui/enclaves/components/widgets/RunViaCIButton.tsx]: is there any way to display the error or rename to `An error occurred starting the enclave`

**Review by tedim52 — APPROVED** [review]
demo isn't loading for some reason

**Comment by h4ck3rk3y** [comment]
https://github.com/kurtosis-tech/kurtosis/assets/6670312/d4545a06-6d27-47fa-9b9f-790c963380fc

**Pull request merged**